web/src/components/MapView.tsx

```diff
@@ -1,4 +1,4 @@
-import * as mapLibreGl from 'maplibre-gl'
+import maplibregl from 'maplibre-gl'
```

Comment:
the exposed interfaces didn't work properly f.e. the `setRTLPlugin` couldn't be found in ts-check